tests/Endpoints/BatchesTest.php

```diff
@@ -36,7 +36,7 @@ public function testGetAllBatchesWithIndexUidFilters(): void
     public function testGetAllBatchesWithTasksFilters(): void
     {
         $tasks = $this->client->getTasks(new TasksQuery())->getResults();
-        $response = $this->client->getBatches((new BatchesQuery())->setUids([$tasks[0]['uid']]));
```

Comment:
Although now this is an object since the task implements `ArrayAccess`, I can still do the `['uid']` right?

Reply:
Yup, but now totally removed